gitops-wg/interested-parties.md

```diff
@@ -3,6 +3,12 @@
 Organization | Person | GitHub Username
 -- | -- | --
 Aivitex | Lothar Schulz
+Akuity | Hong Wang
+Akuity | Jesse Suen
+Akuity | Wojtek Cichoń
+Akuity | Daniel Hoang
+Akuity | Remington Breeze
+Akuity | Yuan Tang
```

Comment:
I was able to find the correct GitHub usernames for everyone but Daniel Hoang.

Everyone added here @wanghong230, @jessesuen, @rbreeze, @terrytangyuan please add your consent (thumbs up, comment, etc), and if you know Daniel's GitHub username we can ping them as well.

```suggestion
Akuity | Hong Wang | wanghong230
Akuity | Jesse Suen | jessesuen
Akuity | Wojtek Cichoń | wojtekidd
Akuity | Daniel Hoang
Akuity | Remington Breeze | rbreeze
Akuity | Yuan Tang | terrytangyuan
```

Reply:
My username is dhpup